template/macros.jinja

```diff
@@ -7,6 +8,8 @@
     Files Switch (TOFS) pattern.
 
     Params:
+      * lookup: key under '<tplroot>:tofs:source_files' to override
+        list of source files
```

Comment:
_If_ we stick with this idea of using `lookup` as the second parameter, then this comment should be moved to under the `source_files` comment below.  The same order for both the parameters and the comments.

Reply:
Thanks.